### ISSUE TITLE
Info on "Windows Update" setting

### ISSUE THREAD

**Issue by ChiragPavecha** [microsoft/service-fabric-explorer#228]
Would be great if we can add Banner/Warning/Link to clusters where Windows Update is still enabled and potentially causing VM’s to reboot unexpectedly.  Link in banner pointing to recommended SF configuration for VMSS OS Updates or POA

**Comment by jeffj6123** [comment]
So I feel like there are a few scenarios we would need to be able to handle for this to work and expose as an API:
- What would we do for linux cluster
- How would this work for on prem clusters

I feel that this would actually be a prime place for SF Observer.  @GitTorre would this be pretty straight forward to implement as an observer?

**Comment by sridmad** [comment]
Agree. Fabric Observer is a better tool to surface this information.
